### ISSUE TITLE
Data issue

### ISSUE THREAD

**Issue by davidlau-apollomit** [GriffinLiang/AISD#18]
Hi @GriffinLiang,

I downloaded the masks, images and dicom files from google drive, but the mask and dicom of the case "0226134" have different number of slices. The mask has 26 slices but dicom only has 16 slices.

**Comment by 3489214402** [comment]
> Hi [@GriffinLiang](https://github.com/GriffinLiang),
> 
> I downloaded the masks, images and dicom files from google drive, but the mask and dicom of the case "0226134" have different number of slices. The mask has 26 slices but dicom only has 16 slices.



> 你好，
> 
> 我从Google Drive下载了遮罩、图片和DICOM文件，但“0226134”机箱的遮罩和DICOM切片数量不同。面具有26片，但Dicom只有16片。
Yes, I think you can refer to the case "0226134" folder in the image and mask.
